math/pi/spigotpi.go

```diff
@@ -0,0 +1,59 @@
+// spigotpi.go
+// description: A Spigot Algorithm for the Digits of Pi
+// details:
+// implementation of Spigot Algorithm for the Digits of Pi - [Spigot algorithm](https://en.wikipedia.org/wiki/Spigot_algorithm)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see spigotpi_test.go
+
+package pi
+
+import "strconv"
+
+func Spigot(n int) string {
+	pi := ""
+	boxes := n * 10 / 3
+	reminder := make([]int, boxes)
```

Comment:
You are missing an `a` in the middle 😃 

```suggestion
	remainders := make([]int, boxes)
```

Reply:
Done